### PR TITLE
Change version for Gtk.StyleContextPrintFlags

### PR DESCRIPTION
**Pull request by gbip**
Change requested in #600.

### CODE REVIEW

**Comment by EPashkin** [comment]
Sorry, but you do wrong changes:
Removing "Gtk.StyleContextPrintFlags" from `generate` array is right,
but change version for `Gtk.StyleContext` is not.
you need add generate flags with version
```
[[object]]
name = "Gtk.StyleContextPrintFlags"
status = "generate"
version = "3.20"
```

**Comment by gbip** [comment]
Woops sorry I went too fast on this one, my bad.

File: Gir.toml [hunk 1]
```diff
@@ -1332,6 +1331,13 @@ status = "generate"
     name = "paint-clock"
     version = "3.8"
 
+
+
```
Comment (EPashkin) [inline Gir.toml]: Please remove 2 empty lines

**Comment by EPashkin** [comment]
Good, only small nit

**Comment by EPashkin** [comment]
@gbip Thanks, just waiting for CI

**Comment by EPashkin** [comment]
@GuillaumeGomez Please take look at this PR.

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**